Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,89 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Threading.Tasks;
 
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
         // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        public IEnumerable<string> CsvRows {
+            get {
+                return File.ReadAllLines("../../../../Assignment/People.csv").ToList().Skip(1); 

```

Comment:
```suggestion
                return File.ReadAllLines("People.csv").ToList().Skip(1); 
```
You should be able to just do this